### ISSUE TITLE
Didn't begin with Zora's Sapphire and kept getting duplicate items and locked out of boss room in Forest Temple

### ISSUE THREAD

**Issue by 03bgood** [AmazingAmpharos/OoT-Randomizer#243]
According to my seed, I was supposed to start out with the Zora's Sapphire but I didn't. Also, the Forest Temple boss key is impossible to obtain, because I need the Mask of Truth to show to the Deku Shrubs in the Lost Woods and it didn't register it the 4th prison key for Gerudo Fortress is also impossible without the Zora's Sapphire since Zelda would've thrown it in the moat, instead of the Ocarina of Time. Therefore the Training Grounds and archery are impossible without freeing all 4 carpenters. What do I do? I can still have access to Ganondorf/Ganon, but I want to get everything.  Also, other chests and item locations had duplicate items, of which I've already obtained; such as the Iron Boots and Biggoron's Sword.

Seed: 141168_ZGD5SANZZO

**Comment by flagrama** [comment]
What emulator are you using?

**Comment by 03bgood** [comment]
It's my Seed number, I don't think it's the emulator. Because it runs fine.

**Comment by flagrama** [comment]
Really? Because it sounds like you're using Project 64
https://github.com/AmazingAmpharos/OoT-Randomizer#known-issues

**Comment by 03bgood** [comment]
No, it's Mupen64Plus FZ on Android. The only problem is the game sometimes crashes if the game's emulation speed is up to 250% and is use for awhile on that setting.

**Comment by flagrama** [comment]
Okay. So, depending on the front-end, by default Mupen64Plus has the same issue as Project64 here: https://wiki.ootrandomizer.com/index.php?title=Project64#Important:_Delete_Your_PJ64_Saves
It is very possible that the Android emulator you are using has this issue by default. You can't delete the in-game save, you have to delete the actual emulator save in the filesystem.

**Comment by 03bgood** [comment]
I have other saves from other OoT Randos, would that have screwed it up?

**Comment by flagrama** [comment]
Yes it can. That's exactly what I'm saying.

**Comment by flagrama** [comment]
Regardless, 1) This is not a rando issue. It is a setup issue. Join the Discord if you need to troubleshoot it further. 2) This is not the active repo for the randomizer.

**Comment by flagrama** [comment]
Lastly you set Item Pool to plentiful. That adds duplicate items to the item pool. That is 100% intended.

**Comment by 03bgood** [comment]
So there was a setting to use duplicate items?

**Comment by flagrama** [comment]
Yes, and you set it.
![image](https://user-images.githubusercontent.com/634079/58677718-12938180-832b-11e9-97f5-7a893b1f7585.png)

**Comment by 03bgood** [comment]
Just started a new file and now I have the Zora's Sapphire. So delete my old file and start over again?

**Comment by flagrama** [comment]
Yes.

**Comment by 03bgood** [comment]
Okay

**Comment by flagrama** [comment]
Again, you should ask these things on the Discord. Not here. This is not what Github Issues is meant for.
